code/modules/clothing/head/helmet.dm

```diff
@@ -374,9 +374,9 @@
 /obj/item/clothing/head/helmet/elder_atmosian
 	name = "Elder Atmosian Helmet"
```

Comment:
```suggestion
	name = "/improper Elder Atmosian Helmet"
```

Reply:
This is mostly so you don't get stuff like 

'Urist McGreyshit throws Elder Atmosian Helmet'

And instead displays as

'Urist McGreyshit throws the Elder Atmosian Helmet'